src/openapi.yaml

```diff
@@ -7592,26 +7608,26 @@ components:
       type: object
       properties:
         previousText:
-          type: string
-          nullable: true
+          type:
+          - string
+          - 'null'
           title: Step previous text [en]
           description: The text that will be displayed in the form step to go to the
             previous step. Leave blank to get value from global configuration.
-          maxLength: 50
         saveText:
-          type: string
-          nullable: true
+          type:
+          - string
+          - 'null'
           title: Step save text [en]
           description: The text that will be displayed in the form step to save the
             current information. Leave blank to get value from global configuration.
-          maxLength: 50
         nextText:
-          type: string
-          nullable: true
+          type:
+          - string
+          - 'null'
```

Comment:
the `maxLength` has disappeared here 🤔 

perhaps we should not opt into OAS 3.1 yet?

Reply:
Yes might be safer: 3.1 support is a couple weeks old, and as you can see there are still issues on the spectacular side